content/en/docs/tasks/traffic-management/tcp-traffic-shifting/gtwapi_test.sh

```diff
@@ -18,7 +18,7 @@
 source "tests/util/gateway-api.sh"
 install_gateway_api_crds
 
-# @setup profile=none
+# @setup profile=demo
 source "content/en/boilerplates/snips/gateway-api-gamma-support.sh"
 bpsnip_gateway_api_gamma_support_enable_alpha_crds
```

Comment:
This line is probably going to mess up the install for following tests. It changes the install to `minimal`.

Reply:
Looks like demo passed...

Reply:
I think it works because the tcp-traffic-shifting test is the last one run, just lucky. It's definitely changing the install profile from demo -> minimal so any test that runs after this one and depends on demo features will break.